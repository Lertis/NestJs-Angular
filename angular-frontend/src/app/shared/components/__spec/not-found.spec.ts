import { ComponentFixture, TestBed, async } from '@angular/core/testing';
import { NotFoundComponent } from '../not-found/not-found.component';
import {} from 'jest';

describe('[Component]: NotFoundComponent', () => {
	let component: NotFoundComponent;
	let fixture: ComponentFixture<NotFoundComponent>;

	beforeEach(async(() => {
		TestBed.configureTestingModule({
			declarations: [NotFoundComponent],
		}).compileComponents();
	}));

	beforeEach(() => {
		fixture = TestBed.createComponent(NotFoundComponent);
		component = fixture.componentInstance;
		fixture.detectChanges();
	});

	it('component should be created', () => {
		expect(component).toBeTruthy();
		expect(component).toMatchSnapshot();
	});
});
