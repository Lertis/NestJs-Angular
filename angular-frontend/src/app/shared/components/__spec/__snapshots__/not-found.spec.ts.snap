// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Component]: NotFoundComponent component should be created 1`] = `NotFoundComponent {}`;
